Networking/NetworkingTests/Remote/StripeRemoteTests.swift

```diff
@@ -327,7 +327,7 @@ final class StripeRemoteTests: XCTestCase {
         // Then
         XCTAssertTrue(result.isSuccess)
         let account = try result.get()
-        XCTAssertEqual(account.isLiveAccount, true)
+        XCTAssertEqual(account.isLiveAccount, false)
```

Comment:
This will have to be reverted when we remove the hardcoded properties in the StripeAccount object, after the extension is updated